modules/ossm-cert-manager-installation.adoc

```diff
@@ -60,6 +52,20 @@ spec:
     secretName: root-ca
 ----
 +
+====
+[NOTE]
+The namespace of the `selfsigned-root-issuer` issuer and `root-ca` certificate is `cert-manager` because `root-ca` is a cluster issuer, so the cert-manager looks for a referenced secret in its own namespace. Its own namespace is `cert-manager` in the case of the {cert-manager-operator}.
+====
+
```

Comment:
**PEER REVIEWER** I copied this note from [Installing cert-manager](https://docs.openshift.com/container-platform/4.13/service_mesh/v2x/ossm-security.html#ossm-cert-manager-installation_ossm-security).